core/cat/routes/plugins.py

```diff
@@ -40,6 +40,8 @@ async def get_available_plugins(
         # get manifest
         manifest = deepcopy(p.manifest) # we make a copy to avoid modifying the plugin obj
         manifest["active"] = p.id in active_plugins # pass along if plugin is active or not
+        manifest["used_hooks"] = [{ "name": hook.name, "priority": hook.priority } for hook in p.hooks]
+        manifest["used_tools"] = [tool.func.__name__ for tool in p.tools]
```

Comment:
Also added tools, should the entry be called `used_tools`, `tools` or `created_tools`?